automatic/autohotkey.install/tools/chocolateyInstall.ps1

```diff
@@ -1,21 +1,55 @@
 ﻿$ErrorActionPreference = 'Stop'
 
 $toolsDir = Split-Path $MyInvocation.MyCommand.Definition
+$installerv1File = 'AutoHotkey_1.1.36.02_setup.exe'
+$installerv2File = 'AutoHotkey_2.0.11_setup.exe'
 
 $pp = Get-PackageParameters
 if (!$pp.DefaultVer){
   $pp.DefaultVer = if ((Get-OSArchitectureWidth 64) -and ($Env:chocolateyForceX86 -ne 'true')) { 'U64' } else { 'U32' }
 }
 
-$packageArgs = @{
+$packageArgsv2 = @{
   packageName    = 'autohotkey.install'
   fileType       = 'exe'
-  file           = Get-Item "$toolsDir\*.exe"
-  silentArgs     = "/S /$($pp.DefaultVer)"
+  file           = Join-Path -Path $toolsDir -ChildPath $installerv2File
+  silentArgs     = "/silent"
   softwareName   = 'AutoHotkey*'
   validExitCodes = @(0, 1223)
 }
-Install-ChocolateyInstallPackage @packageArgs
+
+Install-ChocolateyInstallPackage @packageArgsv2
+
+$installLocation = Get-AppInstallLocation $packageArgsv2.softwareName
+$packageName = $packageArgsv2.softwareName
+$installName = 'AutoHotkey'
+$exePath = Join-Path -Path $installLocation -ChildPath "v2\$installName.exe"
+if ($installLocation) {
+  Write-Host "$packageName installed to '$installLocation'"
+  Register-Application -ExePath $exePath
+  Write-Host "$packageName registered as $installName"
+}
+else {
+  Write-Warning "Can't find $packageName install location"
+}
+
+$packageArgsv1 = @{
+  packageName    = 'autohotkey.install'
```

Comment:
Looking at the output on AppVeyor, do wme maybe want to change the packageName here so it is identifying that it is installing v1 of autohotkey?